packages/cli/src/commands/set-lang/index.ts

```diff
@@ -0,0 +1,38 @@
+import { CommandModule } from 'yargs';
+
+import { z } from 'zod';
+
+import { IContext } from '../../context';
+import { Settings } from '../../settings';
+
+
+const ArgsSchema = z.object({
+    language: z.string(),
+  });
+  
+
+export function SetLang(_: IContext): CommandModule<{}, z.infer<typeof ArgsSchema>> {
+  const language = Settings.load().language ?? '';
+  const currentLanguage = language ? `It is currently set to ${language}.` : '';
+  
+
+  return {
+    command: 'set-lang <language>',
+    describe: `set the programming language for the project (typescript or csharp). ${currentLanguage}`,
+    builder: (b) => {
+      return b
+        .positional('language', {
+          describe: 'programming language to use (typescript or csharp)',
+          type: 'string',
+          choices: ['ts', 'cs'],
```

Comment:
this should support `typescript` and `csharp` as well since those are our command names on `new`

Reply:
added